guidelines/ambassador-allontanamento.md

```diff
@@ -0,0 +1,33 @@
+# Allontanamento Ambassador
+
+Questo documento è stato redatto per fornire una guida al team Ambassador per l'allontanamento di un membro del team.
+
+## Motivi di allontanamento
+
+In tutti i casi sotto elencati, per far partire il sondaggio è sufficiente che un membro del team Ambassador ne faccia richiesta al Communications Group indicando l'evidenza e la regola violata.
+
+Quando viene usato il termine _sondaggio standard infra-gruppo_ si intende un sondaggio con quorum al 50% e maggioranza semplice della durata di 48 ore tra i membri del gruppo di riferimento.
+
+Il mancato raggiungimento del quorum in un sondaggio standard infra-gruppo corrisponderà all'apertura del medesimo sondaggio, con le medesime condizioni, rivolto a tutti i membri del team Ambassador.
+
+Un membro del team Ambassador può essere allontanato per i seguenti motivi:
+
+- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
+- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.
+- Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.
+- Mancata stesura di contenuti propri o review di contenuti altrui nei capitoli assegnati entro 60 giorni dall'ultima stesura e/o review. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per dare una motivazione plausibile. A risposta ricevuta, verrà richiesto un _sondaggio standard infra-gruppo_ per la sua rimozione. Questa regola non si applica ai membri del team Ambassador assegnati a capitoli che non hanno ricevuto alcuna proposta di modifica.
```

Comment:
Non sarebbe meglio introdurre una sorta di "cartellino giallo" (in determinate occasioni, ovvio) per evitare che qualcuno venga espulso così dal nulla?
Servirebbe per evitare che qualcuno violi uno dei punti e subito si becca il rischio espulsione (basterebbe che qualcuno fosse aggiornato con le linee guida redatte dai Communications e dai Relations).

Reply:
L'ho indicata come one-shot perché sta alla sensibilità di ognuno decidere se si sente offeso da quella violazione tanto da segnalarla.. Che dici, meglio specificarlo?

Reply:
Nah, alla fine se ne può benissimo discutere prima di arrivare alla segnalazione.

Reply:
Se ti ho convinto mettila resolved così sappiamo non essere un punto aperto

Reply:
Con infra-gruppo si intende il perimetro ristretto del focus group della persona? O di tutto il gruppo degli Ambassador? Mi sembra sia più logica la seconda.

Altro tema, da capire in caso come gestire il sondaggio tra tutti gli Ambassador - 1 (la persona nominata per espulsione), ovviamente non Telegram.

Inoltre, da capire come comunicare il fatto che si è in corsa per una possibile espulsione e valutare come gestire eventuali reazioni. Cioè: se ti stiamo dicendo che c'è una votazione per espellerti e poi resti, con che morale rimani? Secondo me qui la questione è ancora più spinosa.

Ragionandoci, io **eviterei il sondaggio**. Punterei direttamente sulla chiarezza delle regole e sulla logica dei cartellini come suggerito da Corrado. Se non rispetti le regole, vieni prima avvisato e poi espulso, non dovrebbe esserci un "sondaggio post-regole"

Reply:
> Con infra-gruppo si intende il perimetro ristretto del focus group della persona? O di tutto il gruppo degli Ambassador? Mi sembra sia più logica la seconda.

No, intendo nel working group di riferimento. Se l'utente non rispetta il codice di comportamento, è il Governance a decidere se espellerlo. Se non rispetta le regole di comportamento tra Ambassador del Communications, è il Communications a decidere se espellerlo. Sono i WG di riferimento i responsabili della stesura delle proprie regole, è giusto che lo siano anche nell'applicazione.

> Altro tema, da capire in caso come gestire il sondaggio tra tutti gli Ambassador - 1 (la persona nominata per espulsione), ovviamente non Telegram.

Ovviamente, questo è un tema da trattare dopo aver deciso le linee guida.

> Inoltre, da capire come comunicare il fatto che si è in corsa per una possibile espulsione e valutare come gestire eventuali reazioni. Cioè: se ti stiamo dicendo che c'è una votazione per espellerti e poi resti, con che morale rimani? Secondo me qui la questione è ancora più spinosa. Ragionandoci, io eviterei il sondaggio. Punterei direttamente sulla chiarezza delle regole e sulla logica dei cartellini come suggerito da Corrado. Se non rispetti le regole, vieni prima avvisato e poi espulso, non dovrebbe esserci un "sondaggio post-regole"

Lascio il parere anche ad altri, continuo a credere che il sondaggio sia la cosa migliore, altrimenti ci toccherebbe burocratizzare troppo la cosa.. Se scriviamo tra le regole "Non bisogna bestemmiare" uno potrebbe risponderti che "Dalle sue parti quella parola non è una bestemmia", invece così facciamo in modo che chiunque possa valutare ogni singolo caso nel giusto contesto in cui è calato.

Reply:
@Il-Libro-Open-Source/ambassadors